core/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -498,8 +498,8 @@ impl BroadcastRun for StandardBroadcastRun {
     }
 }
 
-fn should_use_merkle_variant(_slot: Slot, _cluster_type: ClusterType, _shred_version: u16) -> bool {
-    false
+fn should_use_merkle_variant(slot: Slot, cluster_type: ClusterType, shred_version: u16) -> bool {
+    cluster_type == ClusterType::Testnet && shred_version == 28353 && (slot % 19 < 10)
```

Comment:
I guess the idea of checking `shred_version` is that these will automatically get disabled if there is a testnet outage ?

Reply:
It is just an extra sanity check that we are on the right cluster.
It does not matter much and I will remove it once we start the roll out on mainnet.